STEERING-COMMITTEE.md

```diff
@@ -7,51 +7,55 @@ aliases:
   - /contributing/steering-committee/
 ---
 
-The Knative Steering Committee (KSC) is the ultimate authority for the Knative
-project, and governs all aspects of the project.
+The Knative Steering Committee (KSC) defines the governance for Knative
+in pursuit of the goal of an open, vibrant project with a productive
+community, a healthy ecosystem of vendors, and most of all satisfied
+customers running their workloads on Knative clusters.
 
 The governance of Knative is an open, living document, and will continue to
 evolve as the community and project change. We expect over time we will adapt
 the way we run this committee, based on feedback from the community.
 
+- [Mission](#mission)
 - [Charter](#charter)
 - [Delegated authority](#delegated-authority)
 - [Committee Meetings](#committee-meetings)
 - [Committee Mechanics](#committee-mechanics)
 - [Committee Members](#committee-members)
-  - [Allocation of seats](#allocation-of-seats)
+  - [Composition](#composition)
+  - [Nomination of representatives](#nomination-of-representatives)
 - [Decision process](#decision-process)
 - [Changes to the charter](#changes-to-the-charter)
 - [Getting in touch](#getting-in-touch)
 
+## Mission
+
+The Steering Committee's mission is to guide the project such that:
+
+1. Knative is a truly open effort which is inclusive and empowers contributors
+1. Knative core is clearly defined with a conformance suite
+1. Trademark usage is monitored and violations addressed
```

Comment:
I wonder if a more positive statement here (such as "participants have clear rules for usage of the Knative trademark") would make more sense as a mission than focusing on violatons.

Reply:
As an example, everyone could agree that "this is the project that must not be named", which would meet this goal of the mission, but would clearly be a failure for the project.